### PR TITLE
Fabio polish2023 11

### PR DESCRIPTION
**Pull request by jcolomb**

### CODE REVIEW

File: basicframework.md [hunk 1]
```diff
@@ -33,12 +33,12 @@ We usually use 400 mm, and if should not be less than 250 mm to get enough dista
 In this set we will attach the frame (blue marked in the picture above)  to the air table box.
 
 
-- Place two [nut](connectors.yml#5mmNuts){Qty:4} inside each 480 mm rod of the U-formation frameworks on one side which adjacent to the side with nuts already placed in (90 degrees to both rods which can vary in length), both sides work. 
-- Now line these nuts up with the holes inside the long sides of the air table box (480mm) 
+- Place four [nut](connectors.yml#5mmNuts){Qty:8}s inside each 480 mm rod of the U-formation frameworks on one side which adjacent to the side with nuts already placed in (90 degrees to both rods which can vary in length), both sides work.
```
Comment (jcolomb) [inline basicframework.md:36]: 4 or 8? explain

File: basicframework.md [hunk 2]
```diff
@@ -54,17 +54,26 @@ To guarantee a higher stability it's recommended to place a second framework cir
...
 
-- Connects these stuts with the table feets using [nut](connectors.yml#5mmNuts){Qty:8} and [M5x6mm screw](screws.yml#m5x6mm_screw){Qty:8}, placed at the wished height (10 mm works). The angle should be place below the horizontal stuts.
+- Connects these stuts with the table feets using [nut](connectors.yml#5mmNuts){Qty:8} and [M5 screw (6mm)](screws.yml#m5x6mm_screw){Qty:8}, placed at the wished height (10 mm works). The angle should be place below the horizontal stuts.
+
+
+
+
+>i **Note** 
+>i
+>i The following strut is only needed if you want to place the camera on it, it has no stability reasons
```
Comment (jcolomb) [inline basicframework.md:69]:
```suggestion
>i The following strut is only needed if you want to place the camera on it, it does not have a stability purpose
```

File: glue.md [hunk 3]
```diff
@@ -8,10 +8,10 @@
 
 The next step is to glue the central plexiglas box where the air is floating through. Try to keep the predrilled holes free of glue. It's recommended to place the glue on the 8 mm wide sides of the plexiglass.
 
-- Use the [arcylic glue](tools.yml#acrifix_192){qty:some, Cat:tool}[i](glueingadvise.md) to combine the [predrilled 480mmx326mm plexiglass plate](fromstep){Qty:1} with two [predrilled 310x64 plexiglass plate](fromstep){Qty:2}, on the shorter edges. 8 mm should remain on both sides. (The side parts are put in front of the 3mm holes of the top part.)
+- Use the [glue (acrylic glue)](tools.yml#acrifix_192){qty:some, Cat:tool}[i](glueingadvise.md) to combine the [predrilled 480mmx326mm plexiglass plate](fromstep){Qty:1} with two [predrilled 310x64 plexiglass plate](fromstep){Qty:2}, on the shorter edges. 8 mm should remain on both sides. (The side parts are put in front of the 3mm holes of the top part, the predrills must match.)
```
Comment (jcolomb) [inline glue.md:11]:
```suggestion
- Use the [glue (acrylic glue)](tools.yml#acrifix_192){qty:some, Cat:tool}[i](glueingadvise.md) to combine the [predrilled 480mmx326mm plexiglass plate](fromstep){Qty:1} with two [predrilled 310x64 plexiglass plate](fromstep){Qty:2}, on the shorter edges. 8 mm should remain on both sides. (The side parts are put in front of the 3mm holes of the top part, the predrilled holes must match.)
```

File: predrill.md [hunk 4]
```diff
@@ -3,47 +3,47 @@
...
 {{BOM}}
 
-The box will be made of one Top, one bottom, 2 long sides and 2 short sides. In this step, we will drill holes in the Top and long sides plexiglass plates.
+The box will be made of one top, one bottom, 2 long sides and 2 short sides. In this step, we will drill holes in the top, short and long sides plexiglass plates.
+
+
+
+>i **Note** 
+>i
+>i All constructional drawings have a higher resolution if you open them in a new tab.
```
Comment (jcolomb) [inline predrill.md:12]: I do not think so, is it GItbuilding feature I am not aware of ?

**Review by jcolomb — COMMENTED** [review]
@FabioReeh please check and answer

File: glue.md (hunk 3 above)
Comment (FabioReeh) [reply glue.md:11]: pilot holes seems to be the term in english. to use predrills was a bad decision

File: basicframework.md (hunk 1 above)
Comment (FabioReeh) [reply basicframework.md:36]: it is two times 4

File: basicframework.md (hunk 2 above)
Comment (FabioReeh) [reply basicframework.md:69]: suits

File: predrill.md (hunk 4 above)
Comment (jcolomb) [reply predrill.md:12]: if you zoom in your browser, it shows that the resolution has not changed, only the pic is smaller

File: basicframework.md (hunk 1 above)
Comment (jcolomb) [reply basicframework.md:36]: ok

File: predrill.md (hunk 4 above)
Comment (jcolomb) [inline predrill.md:12]:
```suggestion
>i **Note** 
>i
>i Please zoom in (or open in a new window) all constructional drawings if you cannot see them distinctly.
```

**Pull request merged**